src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const correctRobot = {};
+
+  for (const key in robot) {
+    if (correctRobot[robot[key]]) {
```

Comment:
You need to use "hasOwnProperty". This is the second item on the checklist.

Reply:
fixed